.github/workflows/release.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     runs-on: ubuntu-20.04
     env:
       RTE_CONTAINER_IMAGE: quay.io/k8stopologyawareschedwg/resource-topology-exporter
+      E2E_TOPOLOGY_MANAGER_POLICY: SingleNUMANodeContainerLevel
```

Comment:
Something is very suspicious, how the test passed if it is looking for:
https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/88595fb0d865dedcf5a0333b977f1731d2ed0fa7/test/e2e/utils/nodetopology/nodetopology.go#L53

Reply:
Good point, let's look into it a bit deeper

Reply:
Previously, we were reading the expected value from the kubelet config file, so we required no env variable. This obviously works only on kind, we fixed this between 0.2.5 and now.